src/Universe.cpp

```diff
@@ -734,10 +690,12 @@ Lattice::Lattice(const int id, const char* name): Universe(id, name) {
   _offset.setCoords(0.0, 0.0);
 
   /* Default width and number of Lattice cells along each dimension */
-  _num_y = -1;
-  _num_x = -1;
+  _num_x = 0;
```

Comment:
There was a reason that the default number of lattice cells was set to "-1" at initialization, but I can't recall it. Is there a good reason to make it 0.0 now?